conda_forge_tick/migrators/migration_yaml.py

```diff
@@ -500,12 +502,19 @@ def create_rebuild_graph(
                 total_graph.remove_edge(*e)
         # if there isn't a strict dependency or if the feedstock is excluded,
         # remove it while retaining the edges to its parents and children
-        if not inclusion_criteria or node in excluded_feedstocks:
+        if node in excluded_feedstocks:
             pluck(total_graph, node)
```

Comment:
We don't need this pluck or at least it should be moved to the loop below. Hopefully this will make the code more clear.

Reply:
should we clean out these prior to the main loop?

Reply:
I'm not sure. I think what you did is fine.